roles/rhol_crc/tasks/main.yml

```diff
@@ -83,14 +83,14 @@
           ansible.builtin.import_tasks: configuration.yml
 
         - name: Setup RHOL/CRC
-          ansible.builtin.shell:  # noqa risky-shell-pipe
+          ansible.builtin.shell: # noqa risky-shell-pipe
```

Comment:
note that we can remove that `noqa` by adding `set -o pipefail; ` as a first command in that shell.

Reply:
(or we can use ci_script...)

Reply:
True, I did not add these lines, my formatter just fixed the double space(disabled now) 

Regardless I'll fix this in a follow up :)